package.json

```diff
@@ -26,6 +26,10 @@
     "@babel/plugin-proposal-class-properties": "^7.13.0",
     "@babel/preset-env": "^7.14.4",
     "@babel/preset-react": "^7.13.13",
+    "@types/react": "^17.0.11",
+    "@types/react-dom": "^17.0.7",
+    "@types/react-redux": "^7.1.16",
```

Comment:
Have you started using redux?

Reply:
If you are not using @types/react-redux then you should not have it in your repo